datacapture/src/main/java/com/google/android/fhir/datacapture/setup/SdcGlobalConfig.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.fhir.datacapture.setup
+
+object SdcGlobalConfig {
+  /**
+   * SDC Clients should set [AnswerValueSetResolver] for the SDC library to be able access
+   * [org.hl7.fhir.r4.model.Coding]s for the
+   * [org.hl7.fhir.r4.model.Questionnaire.QuestionnaireItemComponent.getAnswerValueSet].
```

Comment:
I don't think the clients `should` set the resolver. They `may` set the resolver. It's not a requirement.

```suggestion
   * An [ExternalAnswerValueSetResolver] may be set to provide answer options dynamically for `choice` and `open-choice` type questions.
```

Reply:
Done